boards/arm/nrf9160dk_nrf52840/doc/index.rst

```diff
@@ -1,6 +1,6 @@
 .. _nrf9160dk_nrf52840:
 
-nRF9160 DK - NRF52840
+nRF9160 DK - nRF52840
```

Comment:
You will need to do this for all Nordic DKs, I think we used this format for all of them.

Reply:
No, the other DKs don't specify the chip, at least not in the title.